lib/presentation/pages/social_sign_up_screen/social_sign_up_layout.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:askm/core/context_extensions.dart';
+import 'package:askm/generated/assets/assets.gen.dart';
+import 'package:askm/presentation/tokens/spacing.dart';
+import 'package:askm/presentation/widgets/background_widget.dart';
+import 'package:askm/presentation/widgets/custom_button.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class SocialSignUpLayout extends StatefulWidget {
+  const SocialSignUpLayout({super.key});
+
+  @override
+  State<SocialSignUpLayout> createState() => _SocialSignUpLayoutState();
+}
+
+class _SocialSignUpLayoutState extends State<SocialSignUpLayout> {
+  static const upperFlex = 5;
+  final appLogo = SvgPicture.asset(Assets.images.logo);
+  final googleIcon = SvgPicture.asset(Assets.images.googleIcon);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: BackgroundWidget(
+        child: Center(
+          child: Column(
+            children: [
+              const Spacer(flex: upperFlex),
+              appLogo,
+              Text(
+                context.s.askm,
+                style: const TextStyle(fontSize: Spacings.xxL),
+              ),
+              Text(
+                context.s.whether,
+                style: const TextStyle(fontSize: Spacings.L),
+              ),
+              Text(
+                context.s.comfort,
+                style: const TextStyle(fontSize: Spacings.L),
```

Comment:
create an application in the core theme/palette/light_palette and textStyles 

And take out all the styles for each component separately, and use colors from there too